src/azure-cli/azure/cli/command_modules/keyvault/_params.py

```diff
@@ -87,6 +87,7 @@ class CLIJsonWebKeyOperation(str, Enum):
         c.argument('sku', arg_type=get_enum_type(SkuName, default=SkuName.standard.value))
         c.argument('no_self_perms', arg_type=get_three_state_flag(), help="Don't add permissions for the current user/service principal in the new vault.")
         c.argument('location', validator=get_default_location_from_resource_group)
+        c.argument('enable_soft_delete', arg_type=get_three_state_flag(), help='Enable vault deletion recovery for the vault, and all contained entities. If omitted, assume true as default value.')
```

Comment:
Where is this variable used? I can't find usage in this PR.

Reply:
It is not used. I added it to separate the help message of `--enable-soft-delete` in `keyvault create` from that in `keyvault update`.